### ISSUE TITLE
[Issue] The "send to printer" function does not work as expected. "Send to Printer" not working

### ISSUE THREAD

**Issue by BambuLabDiscord** [bambulab/BambuStudio#889]
When trying to send a plate or multiple plates to my X1-C, I always get the error message "Send to Printer failed. Please try again".

**Comment by miloutintin** [comment]
I, too, get this almost every time.  Usually one retry is needed, but sometimes several retries is required before it will load to the cloud.

**Comment by UndeadKernel** [comment]
I reported this issue through Discord.
In my case, no matter how many times I try to "Send", I always get an error message.

**Comment by UndeadKernel** [comment]
I'm still observing this issue.
I have now tried using a different windows machine in the same LAN, and I have deactivated the windows firewall. Still, I cannot send anything to the printer.

Is there an easy way to debug what the problem could be?

**Comment by james7360** [comment]
I found a solution on the bambulab forums by obicom66. 

"Unfortunately I can’t see the message but on the beginning I got an error in regards of wrong access key. Logoff & Login again to the PRINTER (not Bambu Studio) solved the issue for me. Now I can use the send option without any problem."

This worked for me just re logged in to the printer using the qr code in the app and then restarted Bambu Studio.
